### PR TITLE
Add Vega.xyz protocol support

### PR DESCRIPTION
**Pull request by ConnorChristie**
Vega.xyz is a decentralized trading platform that is currently still in development and is only running a testnet.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -446,6 +447,7 @@
                   <goal>jar</goal>
                 </goals>
                 <configuration>
+                  <javadocExecutable>${java.home}/bin/javadoc</javadocExecutable>
```
Comment (walec51) [inline pom.xml:450]: this should not be required

File: xchange-examples/src/main/java/org/knowm/xchange/examples/vega/VegaExampleUtils.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package org.knowm.xchange.examples.vega;
+
+public class VegaExampleUtils {
```
Comment (walec51) [inline xchange-examples/src/main/java/org/knowm/xchange/examples/vega/VegaExampleUtils.java:3]: can be removed

File: xchange-vega/src/main/java/org/knowm/xchange/vega/service/VegaMarketDataService.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+        return currencyPairs;
+    }
+
+    public List<CurrencyPair> getExchangeSymbols() {
+        return new ArrayList<>(getCurrencyPairs().keySet());
+    }
+
+    @Override
+    public Ticker getTicker(CurrencyPair currencyPair, Object... args) throws IOException {
```
Comment (walec51) [inline xchange-vega/src/main/java/org/knowm/xchange/vega/service/VegaMarketDataService.java:47]: please try to implement error mapping as described in: https://github.com/knowm/XChange/wiki/New-Implementation-Best-Practices#step-7

**Comment by earce** [comment]
Are you still pursuing this?

**Pull request closed without merging**